styles/info.css

```diff
@@ -127,3 +127,21 @@
   font-size: 12px;
   text-align: center;
 }
+
+@media only screen and (min-width: 1090px) {
+  .info-section {
+    display: grid;
+    grid-template-columns: repeat(3, 0fr);
+    column-gap: 1.5rem;
+    justify-content: center;
+  }
+}
+
+@media only screen and (max-width: 1090px) and (min-width: 768px) {
+  .info-section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    margin: 0 auto;
+    row-gap: 32px;
```

Comment:
- [ ] Please use the same proportioning for this section of cards. Each card should have the same layout for the title and should have more padding on the list underneath:
yours:
![image](https://user-images.githubusercontent.com/91486791/236192959-d98ff5f3-3691-411f-8309-816cb02ed969.png)
figma: 
![image](https://user-images.githubusercontent.com/91486791/236193040-7c2703d9-4570-4882-a8c8-7fdc70e80722.png)